code/code/disc/disc_deikhan.cc

```diff
@@ -149,8 +149,6 @@ int TBeing::doLayHands(const char *arg)
     return FALSE;
   }
   one_argument(arg, name_buf, cElements(name_buf));
-  if (!*name_buf)
-    vict = this;
```

Comment:
I guess I'm replying twice... I'd guess that get_char_room_vis(), on the next line, must handle self, since that line unconditionally overwrites vict.